### PR TITLE
Added documentation for the flag override-existing

### PR DESCRIPTION
**Pull request by gal-forer**
## Related Issues
https://jira-hq.paloaltonetworks.local/browse/XSUP-25880

## Description
Added documentation for override-existing flag for upload

### CODE REVIEW

**Comment by gal-forer** [comment]
I'm fixing the potry now, ignore it @DeanArbel

File: demisto_sdk/__main__.py [hunk 1]
```diff
@@ -1352,7 +1352,9 @@ def format(
 @click.option(
     "--override-existing",
     is_flag=True,
-    help="If true will skip override confirmation prompt while uploading packs.",
+    help="This value(True/False) determine if the user should be presented a confirmation prompt when "
```
Comment (dorschw) [inline demisto_sdk/__main__.py:1355]: please ask @ShirleyDenkberg to review this

File: demisto_sdk/__main__.py [hunk 2]
```diff
@@ -1352,7 +1352,9 @@ def format(
...
+         "attempting to upload a content pack that is already installed on the XSOAR server. This allows the upload "
```
Comment (ShirleyDenkberg) [inline demisto_sdk/__main__.py:1356]:
```suggestion
         "attempting to upload a content pack that is already installed on the Cortex XSOAR server. This allows the upload "
```

File: demisto_sdk/__main__.py (hunk 1 above)
Comment (ShirleyDenkberg) [inline demisto_sdk/__main__.py:1355]:
```suggestion
    help="This value (True/False) determines if the user should be presented with a confirmation prompt when "
```

File: demisto_sdk/commands/upload/README.md [hunk 3]
```diff
@@ -71,6 +71,10 @@ Uploading classifiers to Cortex XSOAR is available from version 6.0.0 and up.
 
     in case a pack was passed in the -i argument and -z is used, DIRECTORY_FOR_THE_ZIP is where to store the zip after creation.
 
+* **--override-existing**
+
+    If true will skip override confirmation prompt while uploading packs.
```
Comment (ShirleyDenkberg) [inline demisto_sdk/commands/upload/README.md:76]:
```suggestion
    If true, will skip the override confirmation prompt while uploading packs.
```

**Comment by ShirleyDenkberg** [comment]
@dorschw @DeanArbel Doc review completed.

**Review by DeanArbel — APPROVED** [review]

**Pull request merged**